nixos/modules/services/web-apps/nextcloud.md

```diff
@@ -17,11 +17,11 @@ and optionally supports
 
 For the database, you can set
 [`services.nextcloud.config.dbtype`](#opt-services.nextcloud.config.dbtype) to
-either `sqlite` (the default), `mysql`, or `pgsql`. For the last two, by
-default, a local database will be created and nextcloud will connect to it via
-socket; this can be disabled by setting
+either `sqlite` (the default), `mysql`, or `pgsql`. For the last two a local
+database may be created by setting
 [`services.nextcloud.database.createLocally`](#opt-services.nextcloud.database.createLocally)
-to `false`.
+to `true`, this will also configure nextcloud connect to it through unix
+socket.
```

Comment:
Isn't an sqlite database created automatically by nextcloud anyways? The sentence "For the last two a local database may be created" is IMHO a bit misleading as it reads - for me at least - as you'd have to do some manual intervention for the sqlite case.

Reply:
I meant to say something in the lines of "you may use this option to create a database", I'll rewrite it to be clearer, thanks.